src/components/TopBar/Breadcrumb.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import type { Account } from '@ledgerhq/live-common/lib/types/account'
+import type { T } from 'types/common'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { createStructuredSelector } from 'reselect'
+import { push } from 'react-router-redux'
+import { translate } from 'react-i18next'
+
+import { accountsSelector } from '../../reducers/accounts'
+import Crumb from './Crumb'
+import Button from '../base/Button'
+
+type Props = {
+  pathname: string,
+  accounts: Account[],
+  push: string => void,
+  t: T,
+}
+
+const mapStateToProps = createStructuredSelector({
+  accounts: accountsSelector,
+})
+
+const mapDispatchToProps = {
+  push,
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  > * {
+    font-family: 'Open Sans';
+    font-weight: 600;
+    font-size: 12px;
+    color: ${p => p.theme.colors.grey};
+  }
+
+  > :first-child {
+    padding: 0px;
+    &:hover {
+      background: transparent;
+      text-decoration: underline;
+    }
+  }
+`
+
+const Separator = styled.div`
+  &::after {
+    content: '/';
+    font-size: 13px;
+    color: ${p => p.theme.colors.fog};
+    padding: 0 15px;
+  }
+`
+
+class Breadcrumb extends PureComponent<Props> {
+  processItemsForDropdown = (items: any[]) =>
+    items.map(item => ({ key: item.id, label: item.id, account: item }))
+
+  render() {
+    const { t, push, pathname, accounts } = this.props
+    let tokenItems
+    let token
+
+    if (!pathname.startsWith('/account/')) return null
+
+    const [, , accountId, tokenId] = pathname.split('/')
```

Comment:
mmh the ordering is weird. `accountId` and then `tokenId`.
it should be `parentAccountId` and `accountId` and then you make a `parentAccount` and `account` (there are no token)

at least, this is the choice we did in other places so we need to be consistent.